plugins/janus_lua.c

```diff
@@ -234,6 +234,12 @@ typedef struct janus_lua_callback {
 	char *argument;
 } janus_lua_callback;
 
+/* helper function to sample the number of occupied slots into Lua stack */
+static void janus_lua_stackdump(lua_State* l)
+{
+    int top = lua_gettop(l);
+    JANUS_LOG(LOG_WARN, "Total in lua stack %d\n", top);
```

Comment:
Is this temporary? How verbose/spammy is this for now? Make it a LOG_HUGE so that people can do LOG_WARN only if they want.